tutorials/Tutorial7_generator.py

```diff
@@ -0,0 +1,75 @@
+        meta={"name": "0"}
+    ),
+    Document(
+        text="""Democratic Republic of the Congo to the south. Angola's capital, Luanda, lies on the Atlantic coast in the northwest of the country. Angola, although located in a tropical zone, has a climate that is not characterized for this region, due to the confluence of three factors: As a result, Angola's climate is characterized by two seasons: rainfall from October to April and drought, known as ""Cacimbo"", from May to August, drier, as the name implies, and with lower temperatures. On the other hand, while the coastline has high rainfall rates, decreasing from North to South and from to , with""",
+    ),
+    Document(
+        text="""Schopenhauer, describing him as an ultimately shallow thinker: ""Schopenhauer has quite a crude mind ... where real depth starts, his comes to an end."" His friend Bertrand Russell had a low opinion on the philosopher, and attacked him in his famous ""History of Western Philosophy"" for hypocritically praising asceticism yet not acting upon it. On the opposite isle of Russell on the foundations of mathematics, the Dutch mathematician L. E. J. Brouwer incorporated the ideas of Kant and Schopenhauer in intuitionism, where mathematics is considered a purely mental activity, instead of an analytic activity wherein objective properties of reality are""",
+        meta={"name": "1"}
+    ),
+    Document(
+        text="""The Dothraki vocabulary was created by David J. Peterson well in advance of the adaptation. HBO hired the Language Creatio""",
+        meta={"name": "2"}
+    ),
+    Document(
+        text="""The title of the episode refers to the Great Sept of Baelor, the main religious building in King's Landing, where the episode's pivotal scene takes place. In the world created by George R. R. Martin""",
+        meta={}
+    )
+]
+
+document_store = FAISSDocumentStore(faiss_index_factory_str="HNSW")
+document_store.delete_all_documents()
+document_store.write_documents(documents)
+
+retriever = DensePassageRetriever(document_store=document_store,
+                                  query_embedding_model="facebook/dpr-question_encoder-single-nq-base",
+                                  passage_embedding_model="facebook/bart-large",
+                                  use_gpu=False, embed_title=True,
+                                  remove_sep_tok_from_untitled_passages=True)
+
+document_store.update_embeddings(retriever=retriever)
+
+docs_with_emb = document_store.get_all_documents()
```

Comment:
I guess everything below here is rather WIP for the current dev...

Reply:
Yes

Reply:
:+1: